src/app/tests/TestCommandInteraction.cpp

```diff
@@ -451,18 +469,18 @@ void TestCommandInteraction::AddInvokeResponseData(nlTestSuite * apSuite, void *
 {
     CHIP_ERROR err = CHIP_NO_ERROR;
 
+    chip::app::ConcreteCommandPath requestCommandPath(1, // Endpoint
+                                                      3, // ClusterId
+                                                      4  // CommandId
+    );
```

Comment:
Other places there is `kTestEndpointId, kTestClusterId, kTestCommandIdWithData`. Don't know which is best (I kinda like seeing the numbers), but perhaps it would help uniformity.